src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.cs

```diff
@@ -119,6 +119,8 @@ internal static void Initialize()
             Instance = new TypeLoaderEnvironment();
             RuntimeAugments.InitializeLookups(new Callbacks());
             s_nativeFormatStrings = new LowLevelDictionary<string, IntPtr>();
+            s_runtimeFieldHandles = new LowLevelDictionary<RuntimeFieldHandleKey, RuntimeFieldHandle>();
```

Comment:
@fadimounir Is there a reason you chose to ignore the code review feedback to make these instance fields like literally everything else in `TypeLoaderEnvironment`?

Cc @davidwrighton